mewarpx/mewarpx/utils_store/parallel_util.py

```diff
@@ -1,15 +1,16 @@
 from mpi4py import MPI as mpi
 import numpy as np
 
-
-from mewarpx.mwxrun import mwxrun
-
-
 comm_world = mpi.COMM_WORLD
 
-def mpiallreduce(data=None, op=None, comm=None):
-    if op is None:
+def mpiallreduce(data=None, opstring="SUM", comm=None):
+    if opstring is None:
```

Comment:
```suggestion
    if opstring is None or opstring == "SUM":
```
I think with how it currently is, not passing an opstring, ie `opstring="SUM"` throws a NotImplementedError